js/backbone.upload-manager.js

```diff
@@ -69,6 +69,13 @@
             // Add local events handlers
             this.bindLocal();
         },
+        // Max Wong: Cannot add same file after cancelled #21
```

Comment:
Haha, it is just my personal practice to add a name if a program is collaborate by multiple programmer.
This is more easy to trace who make this silly code later.

Just remove it when you combine to your version.